### PR TITLE
Run 2to3 on tools directory

### PR DESCRIPTION
**Pull request by gnossen**
This should take care of the easy to fix Python 3 incompatibilities. Generated with:

```
$ find tools -name '*.py' | grep -v "distrib/python" | grep -v "xds_k8s" > todo.csv
$ xargs -i{} 2to3 {} -nw -x future < todo.csv
$ tools/distrib/yapf_code.sh
```

While all of these changes will get us closer to Python 3-compatible state, we should not consider a file fully migrated until all code paths have been tested under Python 3.

### CODE REVIEW

**Comment by gnossen** [comment]
Known issues:
 - https://github.com/grpc/grpc/issues/26019

File: tools/run_tests/python_utils/upload_rbe_results.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import os
 import json
 import sys
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```
Comment (jtattermusch) [inline tools/run_tests/python_utils/upload_rbe_results.py:21]: let's double check the upload_rbe_results.py script actually still works -  if not, we'd silently looks the bigquery data for go/grpc-flaky and we really don't want that.

**Review by jtattermusch — APPROVED** [review]
LGTM after addressing my comments.

**Comment by gnossen** [comment]
@jtattermusch Please take another look.

File: tools/run_tests/xds_k8s_test_driver/config/common.cfg.back [hunk 2]
```diff
@@ -0,0 +1,7 @@
+--resource_prefix=appnet-interop
```
Comment (lidizheng) [inline tools/run_tests/xds_k8s_test_driver/config/common.cfg.back:1]: Accidentally included?

File: tools/run_tests/xds_k8s_test_driver/config/local-dev.cfg.example.back [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Copy to local-dev.cfg
```
Comment (lidizheng) [inline tools/run_tests/xds_k8s_test_driver/config/local-dev.cfg.example.back:1]: Accidentally included?

File: tools/run_tests/xds_k8s_test_driver/config/common.cfg.back (hunk 2 above)
Comment (gnossen) [reply tools/run_tests/xds_k8s_test_driver/config/common.cfg.back]: Yes. Thanks!

File: tools/line_count/yaml2csv.py [hunk 4]
```diff
@@ -1,44 +0,0 @@
-#!/usr/bin/env python
```
Comment (jtattermusch) [inline tools/line_count/yaml2csv.py:1]: note that the line_count kokoro job is still being run continously on master:
https://source.cloud.google.com/results/invocations/6c344a8c-70fa-41cc-b433-70226c33ea99/log
https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:grpc%2Fcore%2Fmaster%2Flinux%2Fgrpc_line_count

let's doublecheck that you aren't deleting any scripts that are required for the continuous run:
https://github.com/grpc/grpc/blob/5cdaec9a4fb9644f4423b5ee180c52747c14d7d2/tools/internal_ci/linux/grpc_line_count.sh

looks like e.g. collect-now.sh is still needed.

**Review by jtattermusch — APPROVED** [review]
LGTM after resolving the line_count comment. Thanks for doing this!

**Pull request merged**